lib/util.js

```diff
@@ -1045,3 +1045,58 @@ process.versions[exports.inspect.custom] =
   (depth) => exports.format(JSON.parse(JSON.stringify(process.versions)));
 
 exports.promisify = internalUtil.promisify;
+
+// Callbackify section
+const CBFY_ERR = 'The last argument to the callbackified function must be a ' +
+                 'callback style function';
+
+
+function callbackifyOnRejected(reason, cb) {
+  // !reason guard inspired by BlueBird https://goo.gl/t5IS6M
+  // Because `null` is a special error value in callbacks which means "no error
+  // occurred". We error-wrap so the callback consumer can distinguish between
+  // "the promise rejected with null" or "the promise fulfilled with undefined".
+  if (!reason) {
+    const newRej = new Error(reason + '');
```

Comment:
This will throw if a Promise is rejected with a `Symbol` value. It would be better to use `new Error(String(reason))`.